### ISSUE TITLE
Inverse of chart-waterfall

### ISSUE THREAD

**Issue by Ziltosh** [Templarian/MaterialDesign#6718]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

I develop a chart based application and I would like to have an icon represent the inverse of the "chart-waterfall" icon already present in the library. Waterfall represent a big red candle, and I need the same but with a big green candle.

Thank you.

### Give us an example of what this icon might look like.

The same like chart-waterfall but "inversed", 4 candles going to bottom and one big candle going to top.

**Comment by Circeus** [comment]
.... If I understand this right, you want the icon, but mirrored horizontally?

**Comment by Ziltosh** [comment]
Yes, but only the candles, not the axises, so, a simple transform does not fit.
